src/styles/typography.scss

```diff
@@ -0,0 +1,46 @@
+@import './fonts.scss';
+
+* {
+  font-family: $open-sans;
+  font-size: 16px;
+  line-height: 22px;
+}
```

Comment:
This isn't typography * is catch all for all elements instead this should be added to base